domains/zakaria-didah.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "hghfth",
+           "email": "zakariadidah@gmail.com",
+           "discord": "788397929308225546"
+        },
+    
+        "record": {
+            "A": ["158.179.216.163"]
```

Comment:
cant visit site (it took too long to respond)

Reply:
sorry. It's fully functional now.

Reply:
cuz we are not lunching the project today. but it's only a week so it's gonna be a nice idea to have your subdomain ready from now.